### PR TITLE
Update code owners, readme and add a new test scenario

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose
$Subject

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,90 @@
...
+
+7. Publish the generated artifacts to the local Ballerina central repository:
+    ```
+    ./gradlew clean build -PpublishToLocalCentral=true
+    ```
+
+8. Publish the generated artifacts to the Ballerina central repository:
+    ```
+    ./gradlew clean build -PpublishToCentral=true
+    ```
```
Comment (daneshk) [inline README.md:73]: we don't need these two. currently, tools are not supported via central, should pack with the distribution.

Comment (sahanHe) [reply README.md]: Updated in [418b6f7](https://github.com/ballerina-platform/persist-tools/pull/12/commits/418b6f743e1545c6ebad9c1a4704f743901efffb)

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# See: https://help.github.com/articles/about-codeowners/
+
+# These owners will be the default owners for everything in the repo.
+*       @daneshk @MadhukaHarith92 @sahanHe
```
Comment (daneshk) [inline .github/CODEOWNERS:7]: Seems like we need commitership to add as code owner

Comment (sahanHe) [reply .github/CODEOWNERS]: Updated in [944beb6](https://github.com/ballerina-platform/persist-tools/pull/12/commits/944beb65051d7602683cebf5d4807854377bdf8a).

**Review by MadhukaHarith92 — APPROVED** [review]

**Pull request merged**